lib/Segment/Client.php

```diff
@@ -177,8 +177,10 @@ private function message($msg, $def = ""){
     if ($def && !isset($msg[$def])) $msg[$def] = array();
     if ($def && empty($msg[$def])) $msg[$def] = (object)$msg[$def];
 
-    if (!isset($msg["context"])) {
```

Comment:
@calvinfo Sure, I'll review and update the code to check msg after a track call! Thanks for the feedback!